src/ert/run_models/manual_update.py

```diff
@@ -0,0 +1,133 @@
+from __future__ import annotations
+
+import functools
+import logging
+from queue import SimpleQueue
+from typing import TYPE_CHECKING
+from uuid import UUID
+
+import numpy as np
+
+from ert.analysis import ErtAnalysisError, smoother_update
+from ert.config import ErtConfig, HookRuntime
+from ert.ensemble_evaluator import EvaluatorServerConfig
+from ert.run_models.run_arguments import ManualUpdateArguments
+from ert.storage import Ensemble, Storage
+
+from ..config.analysis_config import UpdateSettings
+from ..config.analysis_module import ESSettings
+from ..run_arg import create_run_arguments
+from .base_run_model import BaseRunModel, ErtRunError, StatusEvents
+from .event import RunModelStatusEvent, RunModelUpdateBeginEvent
+
+if TYPE_CHECKING:
+    from ert.config import QueueConfig
+
+
+logger = logging.getLogger(__file__)
+
+
+class ManualUpdate(BaseRunModel):
+    """Manual update"""
+
+    def __init__(
+        self,
+        simulation_arguments: ManualUpdateArguments,
+        config: ErtConfig,
+        storage: Storage,
+        queue_config: QueueConfig,
+        es_settings: ESSettings,
+        update_settings: UpdateSettings,
+        status_queue: SimpleQueue[StatusEvents],
+    ):
+        super().__init__(
+            config,
+            storage,
+            queue_config,
+            status_queue,
+            active_realizations=simulation_arguments.active_realizations,
+            total_iterations=1,
+            random_seed=simulation_arguments.random_seed,
+            minimum_required_realizations=simulation_arguments.minimum_required_realizations,
+        )
+        self.prior_ensemble_id = simulation_arguments.ensemble_id
+        self.target_ensemble_format = simulation_arguments.target_ensemble
+        self.ensemble_size = simulation_arguments.ensemble_size
+        self.evaluate_posterior = simulation_arguments.evaluate_posterior
+        self.es_settings = es_settings
+        self.update_settings = update_settings
+        self.support_restart = False
+
+    def run_experiment(
+        self, evaluator_server_config: EvaluatorServerConfig, restart: bool = False
+    ) -> None:
+        log_msg = "Running manual update"
+        logger.info(log_msg)
+        self._current_iteration_label = log_msg
+        ensemble_format = self.target_ensemble_format
+        try:
+            ensemble_id = UUID(self.prior_ensemble_id)
+            prior = self._storage.get_ensemble(ensemble_id)
+            experiment = prior.experiment
+            self.set_env_key("_ERT_EXPERIMENT_ID", str(experiment.id))
+            self.set_env_key("_ERT_ENSEMBLE_ID", str(prior.id))
+            assert isinstance(prior, Ensemble)
```

Comment:
This will throw an `AssertionError` i not true, do we want to catch that as well?

Reply:
This is meant for typing, so _should_ never happen in real life, but on review it is not needed I think.